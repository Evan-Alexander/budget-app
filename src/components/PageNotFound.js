import React from "react";
import { Link } from "react-router-dom";

const PageNotFound = () => (
  <div>
    <p>404</p>
    <p>
      Page not Found! <Link to="/">Go home</Link>
    </p>
  </div>
);
export default PageNotFound;
