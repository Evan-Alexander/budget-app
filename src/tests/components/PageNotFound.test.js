import React from "react";
import { shallow } from "enzyme";
import PageNotFound from "../../components/PageNotFound";

test("Should render PageNotFound component", () => {
  const wrapper = shallow(<PageNotFound />);
  expect(wrapper).toMatchSnapshot();
});
