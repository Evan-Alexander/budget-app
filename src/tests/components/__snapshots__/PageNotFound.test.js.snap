// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render PageNotFound component 1`] = `
<div>
  <p>
    404
  </p>
  <p>
    Page not Found! 
    <Link
      to="/"
    >
      Go home
    </Link>
  </p>
</div>
`;
